OpenEdXMobile/res/layout/fragment_edit_user_profile.xml

```diff
@@ -74,7 +74,8 @@
                     android:drawablePadding="@dimen/x_small_icon_margin"
                     android:text="@string/edit_user_profile_change_photo"
                     android:textColor="@color/disableable_button_text"
-                    android:textSize="@dimen/edx_x_small" />
+                    android:textSize="@dimen/edx_x_small"
+                    android:contentDescription="@string/edit_user_profile_change_photo_description"/>
```

Comment:
Small formatting nit: it looks better to have a space before the `/>` for closing tag.